src/Resources/contao/library/Contao/Controller.php

```diff
@@ -2341,4 +2343,30 @@ protected function createNewVersion($strTable, $intId)
 		$objVersions = new \Versions($strTable, $intId);
 		$objVersions->create();
 	}
+
+	protected static function braceGlob($resource)
+	{
+		if (false === strpos($resource, '/**/') && (defined('GLOB_BRACE') || false === strpos($resource, '{')))
+		{
+			return glob($resource, defined('GLOB_BRACE') ? GLOB_BRACE : 0);
```

Comment:
The `defined()` check here is redundant :)

Reply:
It's not.

Given that `false === strpos($resource, '/**/')` and `false === strpos($resource, '{'))` we still need that check. Even though the flag could be omitted IMO. This comes directly from [Symfonys `GlobalResource`](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Config/Resource/GlobResource.php#L98-L99).

Reply:
Ah, true!